drivers/hmis/lib/form_data/qa_hmis/custom_assessments/HAT.json

```diff
@@ -0,0 +1,65 @@
+{
+  "item": [
```

Comment:
Thanks for adding these for QA!
I'm not seeing anything saved when I submit though. Am I missing something? It also looks like the frontend branch has a different graphql schema, so maybe something is not updated yet.
<img width="666" alt="Screenshot 2024-01-23 at 12 16 45 PM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/cccf2d44-d849-4047-a7b3-8f14edbe0546">

Reply:
Ah oops! I must have forgotten to QA custom assessment/Ce assessment after refactoring. Sorry about that!

fixed in b4e6f6757c623a6271a5283c2c984b66105ad56a
* Added a missing container name for CeAssessment to target the right processor. I believe this is because of the new mapping in RelatedRecordType
* Raise on invalid container name in development or alert to sentry in production. Does this seem okay? I'd like to get an exception if this behavior occurs in the future

Reply:
Yes definitely thanks!